mgmtd/mgmt_txn.c

```diff
@@ -1273,10 +1265,16 @@ static int mgmt_txn_prepare_config(struct mgmt_txn_ctx *txn)
 		goto mgmt_txn_prepare_config_done;
 	}
 
+#ifdef MGMTD_LOCAL_VALIDATIONS_ENABLED
+	if (mm->perf_stats_en)
+		gettimeofday(&txn->commit_cfg_req->req.commit_cfg.cmt_stats
+				      ->validate_start,
+			     NULL);
 	/*
 	 * Validate YANG contents of the source DB and get the diff
 	 * between source and destination DB contents.
 	 */
+	char err_buf[1024] = {0};
```

Comment:
Isn't this against FRR coding convention? Just saying :)

Reply:
which convention?

Reply:
Hmm. I am unable to find the exact reference in the guideline. So its fine. Anyways, I have approved it already.